### PR TITLE
feat(Components/Slider): Implement Action Icon

### PR DESCRIPTION
**Pull request by nicomaligne**
**What is the problem this PR is trying to solve?**
We want the icon on sliderbar to be clickable, to be an action.
Example [here](http://talend.surge.sh/components/?selectedKind=Slider&selectedStory=default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel)
**What is the chosen solution to this problem?**
Add a new props actions that will render specific action if needed. You can still render simple icons.
**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [x] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/src/Slider/Slider.component.js [hunk 1]
```diff
@@ -33,6 +34,31 @@ export function getSelectedIconPosition(icons, value, min, max) {
...
 }
 
+/**
+ * This function allow to get the actions components
+ * @param {array} actions
+ * @param {number} value
+ * @param {number} min
+ * @param {number} max
+ */
+export function getActions(actions, value, min, max) {
```
Comment (acateland) [inline packages/components/src/Slider/Slider.component.js]: Wouldn’t renderActions be a better name ?

**Comment by jmfrancois** [comment]
Why icons are changed in the PR ?

File: packages/components/src/Slider/Slider.component.js (hunk 1 above)
Comment (jare-talend) [inline packages/components/src/Slider/Slider.component.js]: The component has already a event onChange. We have here an implementation very coupled with CMF. 

When we click on an icon, he should just trigger an event onChange with the value.

Comment (nicomaligne) [reply packages/components/src/Slider/Slider.component.js]: Erf didn't know the icon can render a button... 
Love the if that changes the rendering of a component... is so handy.
So yeah you're right for the implem, back to work

Comment (nicomaligne) [reply packages/components/src/Slider/Slider.component.js]: Hmmm another problem is that we have less posibility with the icon than the action. No tooltip for example.

**Pull request merged**